### PR TITLE
Gad 2 2025 latest

### PR DESCRIPTION
**Pull request by Srivatsa19**
## Issue that this pull request solves
Removed the changes in the globals/api/views.py
Closes: # (issue number)

## Proposed changes
Unwanted changes have been reverted
### Brief description of what is fixed or changed
The unwanted changes in globals have been removed and users can only access the urls after logging in
## Types of changes

_Put an `x` in the boxes that apply_

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
https://drive.google.com/drive/folders/1iQBxUOuhojei7qLjph9K1bhximrseokW?usp=sharing
Please attach the screenshots of the changes made in case of change in user interface

## Other information
@varun7singh
@ChaudharyRaman
@prajjwalkapoor
Any other information that is important to this pull request

### CODE REVIEW

File: FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py [hunk 1]
```diff
@@ -223,18 +223,4 @@ class Migration(migrations.Migration):
...
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('name', models.CharField(max_length=200)),
-                ('description', models.CharField(max_length=200)),
-                ('area', models.CharField(max_length=200)),
-                ('requestCreatedBy', models.CharField(max_length=200)),
-                ('engineerProcessed', models.IntegerField()),
-                ('directorApproval', models.IntegerField()),
-                ('deanProcessed', models.IntegerField()),
-                ('status', models.CharField(max_length=200)),
-            ],
```
Comment (Priyansh61) [inline FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py:238]: Any particular reason behind deleting these ?? Like Ig these are auto-generated?

File: FusionIIIT/applications/globals/api/views.py [hunk 2]
```diff
@@ -103,12 +103,6 @@ def profile(request, username=None):
         return Response(data=resp, status=status.HTTP_200_OK)
     elif profile['user_type'] == 'faculty':
         return redirect('/eis/api/profile/' + (username+'/' if username else ''))
-    elif profile['user_type'] == 'staff':
-        resp = {
-            'user' : user_detail,
-            'profile' : profile,
-        }
-        return Response(data=resp, status=status.HTTP_200_OK)
```
Comment (Priyansh61) [inline FusionIIIT/applications/globals/api/views.py:111]: And why are we removing the case when the user is staff?

Comment (Srivatsa19) [reply FusionIIIT/applications/globals/api/views.py:111]: This piece of code was added by the dashboard module member in my local setup and we forgot to revert it back, he told that these changes should not be here and these changes will be reflected in the PR they would raise after their implementation is completed.

File: FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py (hunk 1 above)
Comment (Srivatsa19) [reply FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py:238]: These changes were actually giving errors after running migrations in Anudeep's laptop. The requests model was implemented by us but while making migrations it got added in the 0001_initial.py file. So he told me to remove this piece of code.

**Comment by Priyansh61** [comment]
Ping Integration leads plss to raise a PR for the `latest_staging`. Thanks @Srivatsa19

**Pull request merged**